functions/responsestreaming/StreamBigQuery/Function.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_response_streaming]
+
+using Google.Cloud.Functions.Framework;
+using Microsoft.AspNetCore.Http;
+using System.Threading.Tasks;
+using Google.Cloud.BigQuery.V2;
+using System;
```

Comment:
Remove `using System;` as it's not needed.

Reply:
Kept this since it's used for getting environment variable (GOOGLE_PROJECT_ID)

Reply:
Ah, whoops. In that case, please just re-order the using directives. (If you're using Visual Studio, just Shift-Ctrl-U should sort them.)

Reply:
Done, thank you!